modules/mesos-cluster/main.tf

```diff
@@ -229,3 +229,15 @@ resource "aws_route53_record" "paas-mozilla-org-dns-zone" {
   ttl = 300
   records = ["${aws_elb.mozilla-org-elb.dns_name}"]
 }
+
+resource "aws_cloudwatch_metric_alarm" "cloudwatch-health-host-count-alarm" {
+  alarm_name = "cloudwatch-health-count-${var.environment}"
+  comparison_operator = "GreaterThanToThreshold"
```

Comment:
This will alert us when we have more than 3 healthy masters which is nice to know but could get annoying fast :laughing: Let's go with `LessThanThreshold`